resources/app/src/components/people/DemographicsForm.vue

```diff
@@ -968,19 +968,25 @@ export default {
     mounted() {
 
         {
-            if (!this.uuid) { // Check if uuid prop is not passed
-
+            if (!this.uuid) {
+                // Check if uuid prop is not passed
```

Comment:
I'm not clear why the uuid prop would not be present, but I'll have to take a look at the larger file instead of just this diff. It seems like there should be a less complex way to do this.

Reply:
I did notice that the `addSurvey` function expects a uuid argument, but this is not passed when the function is called from the survey submission button. Maybe this is part of the reason for the problems you were running into?